tests/kani/LayoutRandomization/should_fail.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Kani now has -Zrandomize-layout activated by default.
+// This test only succeeds on the layout of a structure being
+// decided in a deterministic way.
+// With randomization, there is a very very high probability that it fails (1 - (1/2^64))
+
+// kani-flags: --randomize-layout
```

Comment:
Can you simplify this test and just use a specific seed that ensures the test works as expected?

Reply:
I'm not 100% sure that would work
The seed depends on `tcx.def_path_hash`: is that hash going to be the same for every version of the compiler and on every machine? If not, choosing a seed that works right now on my machine is not enough..

Reply:
That's fair. Too bad though.